tests/integration/sunrisa_integration.py

```diff
@@ -265,10 +264,24 @@ def _test_create_entities(sio):
     grow_id = _test_find_all_entities(
         sio, rooms, [rack], [shelf], start_time, end_time, p_level, r_level, b_level
     )
-    print("create_entities_test passed!")
     _test_harvest_grow(sio, grow_id)
+    _test_search_recipes(sio)
+
+    print("create_entities_test passed!")
+
+def _test_search_recipes(sio):
+    flag = []
+    @sio.on("search_recipes_response")
+    def search_recipe_listener(message):
+        assert message["succeeded"] == True
+        print("search_recipe_listener", message)
+        flag.append(True)
 
 
+    sio.emit("search_recipes", {"search_name": "OG"})
```

Comment:
Nit: Don't hardcode this name. Use variable passed in as a parameter.

If you want to, you can still search for a substring rather than the whole variable.

```
strain_name = "OG Kush"
print("first two characters of strain name", strain_name[:2])
```

The above will print `OG`. You can pass that in to `sio.emit`.